README.md

```diff
@@ -1,7 +1,41 @@
-# stack
+# stack(MEVN Stack)
 
 A starter repository for MongoDB, Node.js, and Vue.js, with a local environment based on Docker.
 
+<img width="1848" alt="SCR-20240312-bxuj" src="https://github.com/cihat/cihat.dev/assets/57585087/fe1b82d2-e960-4f9c-87ad-fddc3bfe8511">
+
+Technologies used:
+
+- Backend 
+  - Node.js
+  - Express
+  - MongoDB
+  - Mongoose
+  - Jest
+  - ESLint
+  - Prettier
+  - Husky
+  - Lint-staged
+  - Docker
+
+- Frontend
+  - Vue.js
+  - Pinia
+  - Vite
+  - ESLint
+  - Prettier
+  - Husky
+  
+- DB
+  - MongoDB
+  - Mongoose
+
+- Planing & Documentation & Design & Management 
+  - PlantUML
+  - Trello
+  - iThoughtsX
+  - Figma
```

Comment:
where did these come from :))

Reply:
I've added the tools I use myself; I missed them, so I'm removing them :)

Reply:
I have removed everything except the PlantUML we use.